rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -313,7 +313,7 @@ static bool ppu_break(ppu_thread& ppu, ppu_opcode_t op)
 }
 
 // Set or remove breakpoint
-extern void ppu_breakpoint(u32 addr)
+extern void ppu_breakpoint(u32 addr, bool isAdding)
```

Comment:
Alternatively split into ``ppu_set_breakpoint(u32 addr)``, ``ppu_remove_breakpoint(u32 addr)``
Using a boolean is a bit messy here.

Reply:
I noticed that existed afterwards and deleted it on branch that I'm pushing soon tm

Reply:
So confusing having this branch, a feature branch, and a feature branch merged with yours locally!